usaspending_api/references/tests/test_financial_spending.py

```diff
@@ -33,7 +49,8 @@ def test_award_type_endpoint(client, financial_spending_data):
     resp = client.get('/api/v2/financial_spending/object_class/?fiscal_year=2017&funding_agency_id=1')
     print(resp.data)
     assert resp.status_code == status.HTTP_200_OK
-    assert len(resp.data['results']) > 0
+    assert len(resp.data['results']) == 2
```

Comment:
Thanks for expanding these tests! I think we still need a check for the actual dollar amounts being returned in the response, particularly for a value that's been summed. You may need to one more object class/tas record to support this (_i.e._, make sure that at least one object class/tas combo has more than one record in `FinancialAccountsByProgramActivityObjectClass` to ensure that things are added correctly.